pyproject.toml

```diff
@@ -68,7 +68,10 @@ extend-select = ['Q', 'RUF100', 'C90', 'UP', 'I']
 flake8-quotes = {inline-quotes = 'single', multiline-quotes = 'double'}
 mccabe = { max-complexity = 14 }
 isort = { known-first-party = ['pytest_examples'] }
-target-version = 'py37'
+target-version = 'py38'
+
+[tool.ruff.format]
+quote-style = "single"
```

Comment:
Can we use consistent quotes, double quotes would probably better, but we're using single quotes else where